Helm/opensearch-dashboards/templates/rolebinding.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.rbac.create }}
+kind: RoleBinding
```

Comment:
Very pedantic, but the `kind` and `apiVersion` lines are reversed here compared to the other templates.

Reply:
Makes sense. Will make the change to maintain uniformity